src/main/java/io/strimzi/test/container/StrimziKafkaCluster.java

```diff
@@ -58,7 +58,8 @@ public StrimziKafkaCluster(final int brokersNum, final int internalTopicReplicat
         this.brokersNum = brokersNum;
         this.network = Network.newNetwork();
 
-        this.zookeeper = new StrimziZookeeperContainer()
+        this.zookeeper = new StrimziZookeeperContainer.StrimziZookeeperContainerBuilder()
+            .build()
```

Comment:
This pattern seems to be a bit weird. Why aren't the options inside the builder but outside of it?

Reply:
It was because I did not also extend the Builder instance from `GenericContainer<StrimziKafkaContainerBuilder`. Now it should look like better :)).